tests/framework/common.go

```diff
@@ -215,8 +242,14 @@ func (td *OsmTestData) AreRegistryCredsPresent() bool {
 // Called by Gingkgo BeforeEach
 func (td *OsmTestData) InitTestData(t GinkgoTInterface) error {
 	td.T = t
+	r, err := rand.Int(rand.Reader, big.NewInt(math.MaxUint32))
```

Comment:
How come we're not using `time` to seed the random number generator here? https://stackoverflow.com/a/12321192

Reply:
`rand.crypto` is supposed to be the safest, the reader from crypto is fed entropy from various sources I think.
In fact I was _forced_ to change it to `rand.crypto` as using `math.rand` actually triggers the linter....

Reply:
For non crypto rand generations, using `math.Rand` is perfectly fine, you just need to silence the linter false alarms with `// #nosec` annotation

Reply:
@eduser25 - what is the linter error that you're getting?

Reply:
@shashankram, I see. the linter doesn't actually tell you it is fine for use for non-crypto. 
@michelleN :
In some test file...
```
import (
	"fmt"
	"math/rand"

...

a := rand.Float32()
fmt.Printf("%i", a)
...
```

```
eserra@edu-linux ~/src/osm/tests (metrics) $ golangci-lint run --out-format=github-actions --timeout 5m
::error file=framework/common_profile.go,line=120,col=7::G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
```

Reply:
> @shashankram, I see. the linter doesn't actually tell you it is fine for use for non-crypto.
That's because the linter can't know whether you are using the api for crypto purpose. Doesn't mean you need to use crypto api for non crypto operations, you just need to suppress the false alarm, no other alternative with `gosec` linter enabled.